process2.cpp

```diff
@@ -0,0 +1,138 @@
+#include<iostream>
+//#include<sys/types.h>
+#include<netinet/in.h>
+#include<unistd.h>
+#include<sys/socket.h>
+#include<sys/ioctl.h>
+#include<sys/epoll.h>
+#include<arpa/inet.h>
+#include<vector>
+#include<string.h>
+//#include<sys/socketvar.h>
+//#include<fcntl.h>
+#define SERV_PORT 8000
+int main()
+{
+    std::cout<<"PID process server running: "<<(int)getpid()<<std::endl;
+    int i,listenfd, connfd, sockfd; // varibale for file disriptor
+    int n, client3, client4;
+    //std::vector<int> nosocket; 
+    ssize_t nready, efd, res;
+    char buff[1024], buffer[1024], str[16];
```

Comment:
Don't use specific number, it dose not say any meaning. We can make code easier to read using #define as following example.
#define MAX_BUFFER_SIZE 1024

char buff[MAX_BUFFER_SIZE]

Please find and fix all code segment like this one